Web/Admin.UI/src/app/components/data-acquisition/query-plan-config/query-plan-config.component.ts

```
@@ -0,0 +1,315 @@
+    }
+
+    this.planForm.valueChanges.subscribe(() => {
+      this.formValueChanged.emit(this.planForm.invalid);
+    });
+  }
+
+  ngOnChanges(changes: SimpleChanges) {
+
+    console.log("DataAcquisitionFhirListConfigFormComponent ngOnChanges");
+    console.log(changes);
+    if (changes['item'] && changes['item'].currentValue) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)